modules/shapefile/src/shp-loader.js

```diff
@@ -1,3 +1,4 @@
+import {concatenateChunksAsync} from '@loaders.gl/core';
```

Comment:
Can this be imported from `loader-utils`? Loader modules should not depend on core.

Reply:
`concatenateChunksAsync` doesn't currently exist in `loader-utils`.

Reply:
Should I copy it to `loader-utils`?

Reply:
There's a copy in `loader-utils` which also currently returns a `Uint8Array`:
https://github.com/visgl/loaders.gl/blob/82872c693e9ef86801b11db9f5f81ea623601583/modules/loader-utils/src/lib/binary-utils/memory-copy-utils.js#L20-L30